Rules/AvoidGlobalFunctions.cs

```diff
@@ -0,0 +1,205 @@
+﻿using Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Globalization;
```

Comment:
You need to put this under `#if .. #endif` directives for this to compile against Core CLR. Please have a look at other rules. E.g [AvoidGlobalVars](https://github.com/PowerShell/PSScriptAnalyzer/blob/development/Rules/AvoidGlobalVars.cs)